.github/configs/feature.yaml

```
@@ -12,12 +12,9 @@ develop:
   solc: 0.8.21
 eip7692:
   evm-type: eip7692
-  fill-params: --fork=CancunEIP7692 ./tests/osaka
-  solc: 0.8.21
-eip7692-osaka:
-  evm-type: eip7692-osaka
-  fill-params: --fork=Osaka ./tests/prague ./tests/osaka -k "not slow"
+  fill-params: --fork=Osaka ./tests/osaka
   solc: 0.8.21
+  eofwrap: true
```

We could add these other params as config to `feature.yaml`. Feel free to improve the field names as you please!
```suggestion
  eofwrap:
      enable: true
      fixture_output_path: blockchain_tests/osaka/eofwrap
      ethereum_tests_release_url: https://github.com/ethereum/tests/archive/refs/tags/v14.1.tar.gz
```

Then I think you should be able to access them as `${{ steps.properties.outputs.eofwrap.enable }}`?

---

great idea, will do before I rerun

---

however, if eofwrap field is missing (which we'd like for other configs), will it not crash on some sort of `null.enable` reference? I didn't really get into the specification to figure out how this works, lemme try and see

---

ah, I think nesting a dictionary doesn't work with the `get_release_props` machinery.

---

putting these variables into `env` is good enough maybe? it's all local anyway and also easier to remove once we get rid of eofwrap in the future

---

Ah sorry, I forgot how brittle this mechanism is. Yes, this is fine for now. I've created follow-up issues; we need a better solution for config management: #964, #965.